java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -134,6 +139,61 @@ public void setCommandedState(int s) {
      */
     public static int DELAYED_FEEDBACK_INTERVAL = 4000;
 
+    /**
+     * Duration in Milliseconds of interval between separate Turnout commands.
+     * Note: Not updated after initialisation, which is not an issue as only used in logging. Timing is done via the TurnoutManager.
+     * <p>
+     * Change from e.g. LnTurnout extensions and scripts using {@link #setOutputInterval(int)}
+     */
+    private int TURNOUT_INTERVAL = InstanceManager.turnoutManagerInstance().getOutputInterval(mSystemName);
```

Comment:
This doesn't in general work.  You've got the object initialization asking the instance manager for an object, which in some cases will result in object initialization.

Reply:
This needs to be turnoutInterval, since it’s not a static constant.